src/Http/Controllers/Traits/BreadRelationshipParser.php

```diff
@@ -115,4 +118,16 @@ protected function relationToLink(Model $item, DataType $dataType)
 
         return $item;
     }
+
+    protected function getField($item, $relationMethod)
+    {
+        $relationBuilder = $item->$relationMethod();
+
+        // original field named was used so we'll return that
+        if ($relationBuilder instanceof BelongsTo) return $relationBuilder->getForeignKey();
+
+        // While adding the relationship we used getRelationName() as field name,
+        // so we'll return that now too.
+        if ($relationBuilder instanceof BelongsToMany) return $relationBuilder->getRelationName();
```

Comment:
Awesome thanks! I wasn't sure if it was a conscious decision or not.  I'd just like it to be as complete as possible.  If you don't have time, by all means say so and we'll get someone else to jump in.